sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/PaginatedDocuments.java

```diff
@@ -0,0 +1,143 @@
+package com.microsoft.appcenter.storage.models;
+
+import com.microsoft.appcenter.http.HttpClient;
+import com.microsoft.appcenter.http.ServiceCallback;
+import com.microsoft.appcenter.storage.Utils;
+import com.microsoft.appcenter.storage.client.CosmosDb;
+import com.microsoft.appcenter.utils.async.AppCenterFuture;
+import com.microsoft.appcenter.utils.async.DefaultAppCenterFuture;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+public class PaginatedDocuments<T> implements Iterable<Document<T>> {
+
+    private Page<T> currentPage;
+
+    private TokenResult tokenResult;
+
+    private HttpClient httpClient;
```

Comment:
Is this a gson document? might want to hide from serialization

Reply:
nope~ hmm do not know how to hide it....

Reply:
add transient

Reply:
oki~